qlib/workflow/record_temp.py

```diff
@@ -38,21 +41,22 @@ def generate(self, **kwargs):
         """
         raise NotImplementedError(f"Please implement the `generate` method.")
 
-    def load(self, name, **kwargs):
+    def load(self, name):
         """
         Load the stored records.
 
         Parameters
         ----------
         name : str
             the name for the file to be load.
-        kwargs
 
         Return
         ------
         The stored records.
         """
-        raise NotImplementedError(f"Please implement the `load` method.")
+        # try to load the saved object
+        obj = self.recorder.load_object(name)
```

Comment:
I think putting this method in the base class will raise error when calculating the backtest results, since we are are putting the backtest result under a repository and no relative artifact path is given here.